workflow/rocoto_funcs/ic.py

```diff
@@ -44,7 +44,7 @@ def ic(xmlFile, expdir, do_ensemble=False):
     timedep=f'\n      <timedep><cyclestr offset="{starttime}">@Y@m@d@H@M00</cyclestr></timedep>'
   dependencies=f'''
   <dependency>
-    <and>{timedep}
+    <and>
```

Comment:
Do we no longer need <timedep> for realtime IC and LBC tasks?

Reply:
No need time dependent for IC and LBC. They should only depend on ungrib tasks.

Reply:
Ah, make sense!